### PR TITLE
Markdown file detailing the Strip Checkout Implementation steps

### PR DESCRIPTION
**Pull request by EmaleeSoto**
-Created Checkout markdown file to provide documenatation on the Stripe API and how it is used in Amplify
-File includes implementation steps for Stripe API end points

### CODE REVIEW

**Review by DietBepis1 — CHANGES_REQUESTED** [review]
This is pretty nice, but I think it would be better to add it into the Amplify wiki.

https://github.com/ProgramEquity/amplify/wiki

File: CHECKOUT.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Documentation - Stripe Functions
+
+## Introduction
+
+Amplify uses Stripe to process donations for non-profit organizations. Use these API endpoints to store and complete transactions.
```
Comment (Alex-is-Gonzalez) [inline CHECKOUT.md:5]: Amplify uses Stripe to process donations for non-profit organizations.  Amplify uses these API endpoints to store and complete transactions.

**Review by Alex-is-Gonzalez — COMMENTED** [review]
There is just a small Typo in the start of the MD file. Once that is changed, I will merge it

**Review by Alex-is-Gonzalez — APPROVED** [review]
Thank you for creating the checkout.md

**Pull request merged**